### PR TITLE
Open New Terminal -> Open New External Terminal

### PR DESCRIPTION
**Pull request by solomatov**
Open New Terminal command palette action is confusing since people expect it to open internal terminal, but it opens external terminal instead. I changed the title to make it less confusing.

Also, what do you think about assigning it to Cmd+Shift+` on the Mac so that we have some kind of uniformity in shortcuts?

### CODE REVIEW

File: src/vs/workbench/contrib/externalTerminal/browser/externalTerminal.contribution.ts [hunk 1]
```diff
@@ -130,13 +130,13 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
...
 		id: OPEN_NATIVE_CONSOLE_COMMAND_ID,
-		title: { value: nls.localize('globalConsoleAction', "Open New Terminal"), original: 'Open New Terminal' }
+		title: { value: nls.localize('globalConsoleAction', "Open New External Terminal"), original: 'Open New External Terminal' }
 	}
 });
 
 const openConsoleCommand = {
 	id: OPEN_IN_TERMINAL_COMMAND_ID,
-	title: nls.localize('scopedConsoleAction', "Open in Terminal")
+	title: nls.localize('scopedConsoleAction', "Open in External Terminal")
```
Comment (Tyriar) [inline src/vs/workbench/contrib/externalTerminal/browser/externalTerminal.contribution.ts]: I think this one actually opens in the integrated terminal by default unless you've changed `terminal.explorerKind`

**Review by Tyriar — CHANGES_REQUESTED** [review]
👎 to adding cmd+shift+` since ctrl+shift+` is a companion to ctrl+`, and cmd+` can't be added as it's an OS-level keybinding.

File: src/vs/workbench/contrib/externalTerminal/browser/externalTerminal.contribution.ts (hunk 1 above)
Comment (solomatov) [reply src/vs/workbench/contrib/externalTerminal/browser/externalTerminal.contribution.ts]: Ok. You are right. Sorry.

**Review by Tyriar — APPROVED** [review]
Thanks!

**Pull request merged**